includes/definitions/hatteras.yaml

```diff
@@ -0,0 +1,58 @@
+os: hatteras 
+text: 'Hatteras Overdue DSLAM'
+type: dslam
```

Comment:
`type: dslam` does not exist. 
`type: network` is what should be deployed for a network equipment.

Reply:
fixed